yt_dlp/extractor/redgifs.py

```diff
@@ -58,8 +58,13 @@ def _parse_gif_data(self, gif_data):
         }
 
     def _call_api(self, ep, video_id, *args, **kwargs):
+        headers = {'authorization': 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiIxODIzYzMxZjdkMy03NDVhLTY1ODktMDAwNS1kOGU4ZmUwYTQ0YzIiLCJleHAiOjE2NjM2NzczMjEsInN1YiI6ImNsaWVudFwvMTgyM2MzMWY3ZDMtNzQ1YS02NTg5LTAwMDUtZDhlOGZlMGE0NGMyIiwic2NvcGVzIjoicmVhZCIsInJhdGUiOi0xfQ.uod5xbnhPbdlOb5UTdHPFls3ZtwxKOjkHtN2rpU8h6Y',
+                   'x-customheader': f'https://www.redgifs.com/watch/{video_id}',
+                   'referer': 'https://www.redgifs.com/',
+                   'origin': 'https://www.redgifs.com',
+                   'content-type': 'application/json'}
```

Comment:
* You sure all of these are needed?
* I think this should be a class var.

* Also follow code guidelines:
```suggestion
        headers = {
            'authorization': 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiIxODIzYzMxZjdkMy03NDVhLTY1ODktMDAwNS1kOGU4ZmUwYTQ0YzIiLCJleHAiOjE2NjM2NzczMjEsInN1YiI6ImNsaWVudFwvMTgyM2MzMWY3ZDMtNzQ1YS02NTg5LTAwMDUtZDhlOGZlMGE0NGMyIiwic2NvcGVzIjoicmVhZCIsInJhdGUiOi0xfQ.uod5xbnhPbdlOb5UTdHPFls3ZtwxKOjkHtN2rpU8h6Y',
            'x-customheader': f'https://www.redgifs.com/watch/{video_id}',
            'referer': 'https://www.redgifs.com/',
            'origin': 'https://www.redgifs.com',
            'content-type': 'application/json'
        }
```